python/ray/worker.py

```diff
@@ -1661,7 +1661,7 @@ def kill(actor):
                          "Got: {}.".format(type(actor)))
 
     worker = ray.worker.get_global_worker()
-    worker.core_worker.kill_actor(actor._ray_actor_id)
+    worker.core_worker.kill_actor(actor._ray_actor_id, False)
```

Comment:
Should we expose an option to this `kill` method? It can default to False.

Reply:
@edoakes just saw your comment above. I think adding an option here makes more sense. 

Because, 1) having two different APIs (`kill` and `__ray_terminate__`) is confusing to users. 2) `__ray_terminate__` is also an actor method, if the actor is hanging, it won't take effect.

Reply:
I agree that having `ray.kill` and `actor.__ray_terminate__` is confusing. They have similar effects but with different approaches. We can combine them into one API.

Reply:
Completely agree - ray.kill with a "force kill" flag seems best to me